### DISCUSSION TITLE
alist-web failed to run

### DISCUSSION THREAD

**Discussion by fdddf** [AlistGo/alist#4765] [General]
I'm trying to clone alist and alist-web, after yarn dev runs correctly, it reports "System error: TypeError: route.outlet is not a function" on the web, how to fix this

**Comment by fdddf** [comment]
I tried pnpm like this https://anwen-anyi.github.io/index/build/linux.html, it runs successfully. 
This issue could be ignored.

**Reply by anwen-anyi** [reply]
Has it been resolved?

**Reply by Panici4** [reply]
I had the same problem and cannot be solved by pnpm.

**Reply by loto12345** [reply]
I had the same issue. please follow official steps to git clone & pnpm build.
I think the git clone **--recurse-submodules** https://github.com/alist-org/alist-web.git could solve.
